extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -81,6 +81,7 @@ export const UIType = {
  *    supportedbrowserstate: boolean,
  *    systemuiisvisiblestate: boolean,
  *    uistate: !UIType,
+ *    actionswhitelist: !Array<{tagOrTarget: string, method: string}>,
```

Comment:
Nit: camelCase is easier to read.

Reply:
Filed #18870
Fun fact: a coupe weeks ago, I actually spent half an hour debugging the accesstate with 2 s instead of 3